examples/favorites_advanced/getx_favorites_advanced/lib/feature_puppy/search/views/search_page.dart

```diff
@@ -1,30 +1,47 @@
-import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
 
-import 'package:getx_favorites_advanced/feature_puppy/favorites/controllers/favorite_puppies_controller.dart';
+import 'package:favorites_advanced_base/core.dart';
+import 'package:favorites_advanced_base/resources.dart';
 
-class SearchPage extends StatelessWidget {
+import 'package:getx_favorites_advanced/feature_puppy/controllers/puppy_extra_details_controller.dart';
+import 'package:getx_favorites_advanced/feature_puppy/controllers/puppy_list_controller.dart';
+import 'package:getx_favorites_advanced/feature_puppy/controllers/puppy_manage_controller.dart';
+
+class SearchPage extends GetView<PuppyListController> {
   @override
-  Widget build(BuildContext context) => Container(
-      constraints: BoxConstraints(minWidth: MediaQuery.of(context).size.width),
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.center,
-        crossAxisAlignment: CrossAxisAlignment.center,
-        children: [
-          const Text(
-            'SEARCH PAGE',
-            style: TextStyle(fontSize: 20),
+  Widget build(BuildContext context) => Scaffold(
+        body: controller.obx(
+          (puppies) => RefreshIndicator(
+            onRefresh: () async {
+              await controller.onReload();
+            },
+            child: ListView.builder(
+              padding: const EdgeInsets.only(bottom: 67),
+              itemCount: puppies.length,
+              itemBuilder: (context, index) => Obx(
+                () {
+                  final Puppy item = puppies[index];
+                  return PuppyCard(
+                    key: Key('${Keys.puppyCardNamePrefix}${item.id}'),
+                    puppy: item,
+                    onVisible: (item) => Get.find<PuppyExtraDetailsController>()
+                        .fetchExtraDetails(item),
+                    onCardPressed: (item) {},
+                    onFavorite: (item, isFavorite) {
+                      Get.find<PuppyManageController>()
+                          .markAsFavorite(puppy: item, isFavorite: isFavorite);
+                    },
+                  );
+                },
+              ),
+            ),
           ),
-          ElevatedButton(
-            onPressed: Get.find<FavoritePuppiesController>().incrementCount,
-            child: const Text('Increment Favorites'),
+          onError: (error) => ErrorRetryWidget(
+            onReloadTap: () {
```

Comment:
@RomanovaPrime here you can use expression body. Also please check the static analysis is configured properly. This should be caught by it

Reply:
- prefer_expression_function_bodies  is on, but this is not caught